qulice-pmd/src/test/resources/com/qulice/pmd/JunitStaticPublicMethods.java

```diff
@@ -0,0 +1,45 @@
+package foo.test;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class SomeTest {
+
+    public static final InnerClass INNER = new InnerClass(10, 10);
+
+    @BeforeClass
+    public static void beforeClass(){
+        // setup before class
+    }
+
+    @Test
+    public void emptyTest(){
+        //test something
+    }
+
+    @AfterClass
+    public static void afterClass() throws Exception {
+        //tear down after class
+    }
+
+    @Parameterized.Parameters
+    public static Collection parameters() {
+        return Collections.EMPTY_LIST;
+    }
+
+    public static class InnerClass {
```

Comment:
@vkuchyn I suggest to add the inner class in `StaticPublicMethod` instead of Junit special case file.